src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentManagerExtension.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.ContentManagement
+{
+    public static class ContentManagerExtension
+    {
+        public static async Task UpdateAndCreateAsync(this IContentManager contentManager, ContentItem contentItem, VersionOptions options)
```

Comment:
Why this method and not just call both where it's used?

Reply:
We could have done that too. But personally I think it's cleaner/easier that way.

Reply:
It's kind of like why Orchard.DisplayAsync() ?